letsencrypt/data/run.sh

```diff
@@ -46,13 +58,23 @@ echo -e "dns_cloudflare_email = $(bashio::config 'dns.cloudflare_email')\n" \
   "dns_sakuracloud_api_secret = $(bashio::config 'dns.sakuracloud_api_secret')" > /data/dnsapikey
 chmod 600 /data/dnsapikey
 
-# AWS workaround
+# AWS
 if bashio::config.exists 'dns.aws_access_key_id' && bashio::config.exists 'dns.aws_secret_access_key'; then
     AWS_ACCESS_KEY_ID="$(bashio::config 'dns.aws_access_key_id')"
     AWS_SECRET_ACCESS_KEY="$(bashio::config 'dns.aws_secret_access_key')"
 
     export AWS_ACCESS_KEY_ID
     export AWS_SECRET_ACCESS_KEY
+    PROVIDER_ARGUMENTS="--${DNS_PROVIDER}"
+#Google
+elif bashio::config.exists 'dns.google_creds'; then
+    GOOGLE_CREDS="$(bashio::config 'dns.google_creds')"
+
+    export GOOGLE_CREDS
+    PROVIDER_ARGUMENTS=("--${DNS_PROVIDER}" "--${DNS_PROVIDER}-credentials" /ssl/"$GOOGLE_CREDS")
```

Comment:
I think the share folder is a better place? We can also move it from share into `/data` on first start to make sure that it will work later if the user cleanup the share.

Reply:
Updated as requested.